.github/workflows/check-catalyst.yaml

```
@@ -372,6 +373,23 @@ jobs:
         MDD_BENCHMARK_PRECISION=1 \
         pytest demos --nbmake -n auto
 
+    - name: Checkout Lightning-Kokkos repository
+      uses: actions/checkout@v3
+      with:
+        repository: PennyLaneAI/pennylane-lightning-kokkos
+        ref: v0.30.0
+
+    - name: Install PL's lightning.kokkos backend device
+      run: |
+        # Install PL's lightning.kokkos backend device
+        cd pennylane-lightning-kokkos \
+        python3 -m pip install -r requirements.txt \
+        BACKEND="OPENMP" python3 -m pip install -e .
```

```suggestion
        cd pennylane-lightning-kokkos && \
        python3 -m pip install -r requirements.txt && \
        BACKEND="OPENMP" python3 -m pip install -e .
```

Although I am not sure about the variable `BACKEND="OPENMP"` can be declared after &&. Do these need to be run as the same command? Could the following work instead?

```suggestion
        cd pennylane-lightning-kokkos
        python3 -m pip install -r requirements.txt
        BACKEND="OPENMP" python3 -m pip install -e .
```